### DISCUSSION TITLE
Spock 2.4-M4

### DISCUSSION THREAD

**Discussion by leonard84** [spockframework/spock#1932] [Releases]
https://spockframework.org/spock/docs/2.4-M4/index.html

<hr /><em>This discussion was created from the release <a href='https://github.com/spockframework/spock/releases/tag/spock-2.4-M4'>Spock 2.4-M4</a>.</em>

**Comment by sunterry0613** [comment]
Hi 
Regarding version 2.4, the most recent releases are all milestone releases, is there a planned time for the official release?

**Comment by dlehammer** [comment]
Hi @leonard84 ,

I think the release link should point to https://spockframework.org/spock/docs/2.4-M4/release_notes.html ~ aligning with earlier releases 🤓
